### PR TITLE
added dpdt switch

### PR DESCRIPTION
**Pull request by anugcodes**
#142 
Design for the DPDT switch
image

@seveibar , @Timer00 , @imrishabh18  please review this and suggest any changes.

### CODE REVIEW

File: symbols/DPDT_switch_horz.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    // { ...refblocks.top2, labels: ["2"] }, // TODO add more "standard" labels
+    // { ...refblocks.top3, labels: ["3"] }, // TODO add more "standard" labels
+    // { ...refblocks.bottom1, labels: ["4"] }, // TODO add more "standard" labels
+    // { ...refblocks.bottom2, labels: ["5"] }, // TODO add more "standard" labels
+    { ...refblocks.left1, labels: ["1"] }, // TODO add more "standard" labels
+    { ...refblocks.left2, labels: ["4"] }, // TODO add more "standard" labels
+    { ...refblocks.right1, labels: ["3"] }, // TODO add more "standard" labels
+    { ...refblocks.right2, labels: ["2"] }, // TODO add more "standard" labels
+    { ...refblocks.right3, labels: ["6"] }, // TODO add more "standard" labels
+    // { ...refblocks.right4, labels: ["11"] }, // TODO add more "standard" labels
```
Comment (seveibar) [inline symbols/DPDT_switch_horz.ts:25]: lots of ghost code, probably best to remove it to avoid confusion

File: generated/symbols-index.ts [hunk 2]
```diff
@@ -104,6 +106,8 @@ export default {
   "diac_vert": _diac_vert,
   "diode_horz": _diode_horz,
   "diode_vert": _diode_vert,
+  "DPDT_switch_horz": _DPDT_switch_horz,
+  "DPDT_switch_vert": _DPDT_switch_vert,
```
Comment (seveibar) [inline generated/symbols-index.ts:110]: symbols should always be lowercase

**Review by seveibar — COMMENTED** [review]
should this be named like "dpdt_switch_pos1" or something? I'm just thinking there's probably a variant where the switch connects the other way that we'd want to add. Otherwise looks great! Make sure to resolve merge conflicts and see minor comments

**Comment by anugcodes** [comment]
> should this be named like "dpdt_switch_pos1" or something? I'm just thinking there's probably a variant where the switch connects the other way that we'd want to add. Otherwise looks great! Make sure to resolve merge conflicts and see minor comments

I think adding design for both the positions is not necessary as both will serve the same purpose for designing circuits but may be useful for showing the states of the switch during a simulation of the circuit.

**Pull request closed without merging**